daemons/pacemakerd/pacemakerd.c

```diff
@@ -37,6 +38,14 @@ struct {
     gboolean standby;
 } options;
 
+static pcmk__output_t *out = NULL;
+
+static pcmk__supported_format_t formats[] = {
+    PCMK__SUPPORTED_FORMAT_NONE,
+    PCMK__SUPPORTED_FORMAT_TEXT,
+    { NULL, NULL, NULL }
```

Comment:
I think it might be useful to have --features in XML

Reply:
I've added this as a separate patch, since it might be more globally useful than just hiding in pacemakerd.  Also, I wonder if this means I need to add XML schema for it, or if we care to validate this output.